recipes/mbits-args/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class MBitsArgsConan(ConanFile):
+    name = "mbits-args"
+    description = "Small open-source library for program argument parser, inspired by Python's `argparse`, " \
+                  "depending only on the standard library, with C++17 as minimum requirement."
+    homepage = "https://github.com/mbits-libs/args"
+    license = "MIT"
+    topics = ("command-line", "commandline", "commandline-interface",
+              "program-arguments", "argparse", "argparser", "argument-parsing")
+
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    _compilers_minimum_version = {
+        "gcc": "8",
+        "clang": "7.0",
+        "Visual Studio": "16",
+        "apple-clang": "10.0",
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+            compiler = str(self.settings.compiler)
+            runtime = self.settings.compiler.get_safe("runtime")
+            if compiler == "Visual Studio" and runtime and str(runtime)[:2] == "MT":
```

Comment:
You can compare against the `SettingsItem` objects.
```suggestion
            if self.settings.compiler == "Visual Studio" and "MT" in str(self.settings.compiler.runtime):
```

Reply:
Done